src/main/js/field/EndpointButtonField.js

```diff
@@ -144,6 +144,7 @@ class EndpointButtonField extends Component {
 }
 
 EndpointButtonField.propTypes = {
+    id: PropTypes.string,
```

Comment:
Is this property used in this component?

Reply:
Yes line 128 the `{...this.props}` the id of this component is used in the LabeledField.  But I have to double check LabeledField is setting it.